docs/wallet/dlc.md

```diff
@@ -115,71 +71,69 @@ Upon receiving a DLC Accept message from your counter-party, you can use the `Si
 Upon receiving a DLC Sign message from your counter-party, add their signatures to your database using the `Add Sigs` button and paste in the message.
 After doing so you can get the fully signed funding transaction using the `Get Funding Tx` button. This will return the fully signed serialized transaction.
 
-### Step 4: Executing the DLC
-
-#### Execute
-
-You can execute the DLC unilaterally with the `Execute` button which will require the oracle signature.
-This will return a fully signed Contract Execution Transaction for the event signed by the oracle.
-
-#### Refund
-
-If the `refundlocktime` for the DLC has been reached, you can get the fully-signed refund transaction with the `Refund` button.
-
-## Using the CLI
-
-### Step 3: Setup The DLC
+### Using the CLI
 
 #### Creating The Offer
 
 Once these terms are agreed to, either party can call on `createdlcoffer` with flags for each of the fields in the table above. For example:
 
 ```bashrc
-./app/cli/target/universal/stage/bitcoin-s-cli createdlcoffer --oracleInfo 02debeef17d7be7ced0bf346395a5c5c7177491953e91f0af2b098aac5d23cabb1a63752e5a760f47252545b7cda933afeaf06dba3b6c6fd5356781f240c2750 --contractInfo c07803e32c12e100905e8d69fe38ae72f2e7a17eb7b8dc1a9bce134b0cbe920fa0860100000000005c58e41254e7a117ee1db59874f2334facc1576c238c16d18767b47861f93f7c0000000000000000 --collateral 40000 --locktime 1666720 --refundlocktime 1666730 --feerate 3
+./app/cli/target/universal/stage/bitcoin-s-cli createdlcoffer [contractInfo] [collateral] [feerate] [locktime] [refundlocktime]
 ```
 
-This will return a nice pretty-printed JSON offer. To get an offer that can be sent to the counter-party, add the `--escaped` flag to the end of this command.
-
 #### Accepting The Offer
 
 Upon receiving a DLC Offer from your counter-party, the following command will create the serialized accept message:
 
 ```bashrc
-./app/cli/target/universal/stage/bitcoin-s-cli acceptdlcoffer --offer [offer] --escaped
+./app/cli/target/universal/stage/bitcoin-s-cli acceptdlcoffer [offer]
 ```
 
 #### Signing The DLC
 
 Upon receiving a DLC Accept message from your counter-party, the following command will generate all of your signatures for this DLC:
 
 ```bashrc
-./app/cli/target/universal/stage/bitcoin-s-cli signdlc --accept [accept] --escaped
+./app/cli/target/universal/stage/bitcoin-s-cli signdlc [accept]
```

Comment:
so IIRC this can can generate a ton of signatures. How do we handle this from the cli? Do we have the export capability here as well?

Reply:
Added note about the to and from file for cli